actions/java-gradle-build-jib/action.yaml

```diff
@@ -65,20 +75,41 @@ runs:
         gradle-cache: ${{ inputs.gradle-cache }}
         gradle-cache-read-only: ${{ inputs.gradle-cache-read-only }}
 
+    - name: Process inputs
+      id: process-inputs
+      run: |
+        # Extract the image name from the full image name
+        echo "IMAGE_NAME=$(echo "${{ inputs.full-image-name }}" | cut -d ':' -f 1)" >> "$GITHUB_OUTPUT"
+        IFS=',' read -r -a tags <<< "${{ inputs.additional-tags }}"
+        echo "IMAGE_TAGS<<EOF" >> "$GITHUB_OUTPUT"
+        for element in "${tags[@]}"
+        do
+            echo "type=raw,$element" >> $GITHUB_OUTPUT
+        done
+        echo "EOF" >> "$GITHUB_OUTPUT"
+      shell: bash
+
+    - name: Docker metadata
+      id: meta
+      uses: docker/metadata-action@v5
+      with:
+        images: ${{ steps.process-inputs.outputs.IMAGE_NAME }}
+        tags: |
+          type=ref,event=branch
+          type=sha
+          ${{ steps.process-inputs.outputs.IMAGE_TAGS }}
+
     - name: Build Docker image
       run: |
-        ./gradlew ${{ inputs.subproject && format('{0}:', inputs.subproject) || '' }}jibBuildTar \
+        ./gradlew ${{ inputs.subproject && format('{0}:', inputs.subproject) || '' }}jib \
           --info --stacktrace \
-          --image=${{ inputs.image-name }} \
+          --image=${{github.repository}}_${{ github.run_id }}_${{github.run_number}} \
+          -Djib.from.platforms=${{ inputs.platforms }} \
           ${{ inputs.jib-from-image && format('-Djib.from.image={0}', inputs.jib-from-image) || '' }} \
-          -x test \
-          ${{ inputs.gradle-refresh-dependencies == 'true' && '--refresh-dependencies' || '' }}
+          ${{ inputs.class && format('-Djib.container.mainClass={0}', inputs.class) || '' }} \
+          -Djib.to.image=${{ inputs.full-image-name }} \
```

Comment:
According to the input description of `full-image-name` this can have a tag attached as well. Did you make sure the tag here is also published during testing? It might be cleaner only have a single action input for tags, e.g. not allowing tags in the full image name and rename the `additional-tags` to `image-tags` (still as a comma separated list).

Reply:
yes